src/genie/libs/parser/iosxe/tests/ShowWirelessFabricClientSummary/cli/equal/golden_output_expected.py

```diff
@@ -0,0 +1,269 @@
+expected_output = {
+    "number_of_fabric_clients": 33,
+    "clients": {
```

Comment:
`mac_address` shouldn't be duplicated. so, below is better instead of using index `clients`?
```
"mac_address": {
    "58bf.ea71.80f0": {
        "ap_name": "a4-21-cap28",
        "wlan": 19,
        (snip)
```

Reply:
Good idea. Updated code to reflect request.

Reply:
confirmed